build.gradle

```diff
@@ -10,6 +10,13 @@ archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
+repositories {
+    // Add repositories to retrive artifacts from in here.
```

Comment:
```suggestion
    // Add repositories to retrieve artifacts from in here.
```

Reply:
Ah yes I can very much spell, thank you.